storage/include/s3/metrics.hpp

```diff
@@ -35,10 +36,23 @@ struct Metrics {
     // the format is: "PREFIX/BLOCK_ID/KEY", where PREFIX is optional
     if (elems.size() < 2) return;
 
-    lastSavedBlockId_.Get().Set(elems[elems.size() - 2]);
+    uint64_t lastSavedBlockVal = 0;
```

Comment:
Nit: As far as I see, the file uses snake_case, so probably`last_saved_block_val`?
P.S. It is really frustrating that we have both...

Reply:
Actually, it is mixed... so ignore, please.

Reply:
Good point. I fixed the naming in the whole file. It's a new source so it should be styled correctly.